app/models/manageiq/providers/amazon/container_manager/refresh_worker/runner.rb

```diff
@@ -1,2 +1,18 @@
 class ManageIQ::Providers::Amazon::ContainerManager::RefreshWorker::Runner < ManageIQ::Providers::Kubernetes::ContainerManager::RefreshWorker::Runner
+  private
+
+  def filter_notices!(notices)
```

Comment:
Is the caller for this already implemented in the base class?

Reply:
Nope this has a dependent PR

Reply:
I see a caller for `filter_notices` (no-bang), but not `filter_notices!`

Reply:
Yeah I'm thinking of pulling the `Collector#filter_notices` from the base class into the runner, the reason I didn't do this in the collector is by that point we've already started doing a refresh which negates the purpose of reducing churn